### PR TITLE
chore(template): remove inlined ZipFile for all workload Lambda fns

### PR DESCRIPTION
**Pull request by efekarakus**
This PR will be followed-up with:
- Deleting reading the old ZipFiles
- Automatically generating the shas on `svc package` instead of leaving them empty

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by huanjani — APPROVED** [review]
LGTM!

File: internal/pkg/cli/deploy/deploy.go [hunk 1]
```diff
@@ -684,7 +682,7 @@ func (d *workerSvcDeployer) GenerateCloudFormationTemplate(in *GenerateCloudForm
 	return d.generateCloudFormationTemplate(output.conf)
 }
 
-// DeployWorkload deploys a worker servsice using CloudFormation.
+// DeployWorkload deploys a worker service using CloudFormation.
```
Comment (iamhopaul123) [inline internal/pkg/cli/deploy/deploy.go:685]: 🤦

File: internal/pkg/cli/deploy/deploy.go [hunk 2]
```diff
@@ -674,7 +672,7 @@ func (d *workerSvcDeployOutput) RecommendedActions() []string {
 	return recs
 }
 
-// GenerateCloudFormationTemplate genrates a CloudFormation template and parameters for a workload.
+// GenerateCloudFormationTemplate generates a CloudFormation template and parameters for a workload.
```
Comment (iamhopaul123) [inline internal/pkg/cli/deploy/deploy.go:675]: 😮‍💨

File: internal/pkg/deploy/cloudformation/stack/lb_grpc_web_service_integration_test.go [hunk 3]
```diff
@@ -79,28 +76,11 @@ func TestGrpcLoadBalancedWebService_Template(t *testing.T) {
...
 
 		t.Run(testName, func(t *testing.T) {
 			actualBytes := []byte(tpl)
 			// Cut random GUID from template.
 			actualBytes = regExpGUID.ReplaceAll(actualBytes, []byte("RandomGUID"))
-			actualString := string(actualBytes)
-			// Cut out zip file for more readable output
-			actualString = strings.ReplaceAll(actualString, envControllerZipFile, "mockEnvControllerZipFile")
-			actualString = strings.ReplaceAll(actualString, dynamicDesiredCountZipFile, "mockDynamicDesiredCountZipFile")
-			actualString = strings.ReplaceAll(actualString, rulePriorityZipFile, "mockRulePriorityZipFile")
```
Comment (iamhopaul123) [inline internal/pkg/deploy/cloudformation/stack/lb_grpc_web_service_integration_test.go:101]: awesome!

**Review by iamhopaul123 — APPROVED** [review]
🎉

**Pull request merged**